src/System.IO/src/System/IO/TextWriter.cs

```diff
@@ -20,20 +20,8 @@ public abstract class TextWriter : IDisposable
     {
         public static readonly TextWriter Null = new NullTextWriter();
 
-        // This should be initialized to Environment.NewLine, but
-        // to avoid loading Environment unnecessarily so I've duplicated
-        // the value here.
```

Comment:
@jkotas, I've obviously ignored this comment.  Is there a good remaining reason to be concerned about it?